### ISSUE TITLE
Adding more signatures to a database file 

### ISSUE THREAD

**Issue by AnneliektH** [sourmash-bio/sourmash#2904]
Hello! 
I got a database file with signatures, a zip file. Can I add new signatures to this, using sourmash cat? 

I would like to add the same signatures to the manifest file as well, so that I can use it for sourmash fastgather, and use the results as a pick list for sourmash gather.

**Comment by ctb** [comment]
Yes, I believe `sourmash sig cat <new sigs> -o existing-db.zip` should update the `existing-db.zip` file to contain the new signatures! You can check this with `sourmash sig summarize` (to see if the database has increased in number etc.) or `sourmash sig details | grep <name of sig>` to see if a signature has been added.  Please let me know if that doesn't work!

More generally, this will depend on the database type. You can see a list of all available database types [here](https://sourmash.readthedocs.io/en/latest/command-line.html#choosing-signature-output-formats).

* it should work for zip files and directories (`-o dirname/`)
* signature JSON files (`.sig` and `.sig.gz`) will just be overwritten instead
* I'm not actually sure about sqldb...

Now, for fastgather, you should be aware that it dynamically unpacks the zipfile each time, at least at the moment. Which isn't that fast. So I've taken to using the approach listed [over in sourmash-slainte Snakefile](https://github.com/dib-lab/sourmash-slainte/blob/main/Snakefile#L202). The good news for you is that adding signatures in this situation is easy, although it requires a bit of paperwork - you need to drop the new .sig/.sig.gz files into the directory you're using, and add them to the text file listing of what signatures are available.

If that's too much information, I apologize 😁

**Comment by ctb** [comment]
Minor side note: `fastgather` now interacts properly with zipfiles (and has for quite a while, actually) so zip files are the right way to go.
